### DISCUSSION TITLE
Change filter of all Charts present in a Dashboard from a single chart

### DISCUSSION THREAD

**Discussion by kwims** [apache-superset/superset-ui#858] [Questions]
Hi Everyone,

We are trying from a chart that is not filter one, to make certain part of the visualization clickable so that it will update the filter chart and therefore filter all charts present in the dashboard.

Any idea how to achieve this? 

Thanks in advance.

**Comment by amitmiran137** [comment]
Cross filter ability will be available on the upcoming weeks as part of the native filters feature .

**Reply by kwims** [reply]
Wow that's great. Looking forward to it. One last question, Is there already concrete plan when we can expect that?  Are we talking about this year still or January?

**Reply by amitmiran137** [reply]
January is a realistic due date 👌

**Reply by kwims** [reply]
Great, thx a lot. Looking forward to use it.

**Comment by kwims** [comment]
hey guys,
any update on this?

**Comment by junlincc** [comment]
> hey guys,
> 
> any update on this?

Still under construction. Soon.
